### ISSUE TITLE
Buttons being removed from other embeds

### ISSUE THREAD

**Issue by ItsLewizzz** [ygimenez/Pagination-Utils#49]
**Checklist**

Make sure that you've checked all the items below.

- [x] Bot has the following permissions:
  - MESSAGE_ADD_REACTION
  - MESSAGE_EXT_EMOJI
  - MESSAGE_READ/WRITE
  - VIEW_CHANNEL
- [x] If using `JDABuilder.createLight()`, you added the following gateway intents:
    - GUILD_MESSAGES
    - GUILD_MESSAGE_REACTIONS
- [x] If using `.setRemoveOnReact(true)`, you have the following permission:
    - MESSAGE_MANAGE
- [x] PaginationUtils is up-to-date.
- [x] You have activated the library as descripted in the README.

**Library info**

What libraries versions are you using.
- JDA version 5.0.0-beta.10
- Pagination-Utils version 4.0.5b

**Describe the bug**

When I add an ActionRow to other embeds I have in my bot (that are not handled by Pagination-Utils), as soon as I react on any button the whole row is deleted. When the util was disabled my other embed buttons worked fine without being deleted.

**To Reproduce**

1. Create external embed with buttons
2. Interact with one of the buttons
3. The whole action row will be deleted.

**Expected behavior**

Pagination Utils should not be handling other embed button listeners. 

**Screenshots**

With Pagination-Utils enabled on external embed with "buttons" that is handled by myself on my own listener.
![image](https://github.com/ygimenez/Pagination-Utils/assets/29049501/6257ea23-7032-4360-a9b0-2c3ff902b0f1)

With Pagination-Utils disabled (working fine, my buttons show and are usable):
![image](https://github.com/ygimenez/Pagination-Utils/assets/29049501/27663494-ea25-4a48-a0f1-c30848c8ff8b)

**Additional context**

**Comment by ygimenez** [comment]
This happens because the library assumes all button handling will be done by itself (like with `Pages.buttonize`), I'll add a setting to make it ignore buttons not created through its methods on the next update.

Meanwhile, you can manually add the message to the events map through `Pages.getHandler().addEvent(message, null)`, the second argument says non-null but it won't cause errors as it checks whether the consumer is null, just remember to manually remove the event afterwards, else it'll stay until you shutdown the bot (possibly causing a memory leak).

**Comment by ItsLewizzz** [comment]
Is there any ETA for this?

**Comment by ygimenez** [comment]
Sorry, I've been busy with some tasks at work, couldn't touch it yet, but I should be able to deal with it soon.

**Comment by ygimenez** [comment]
I apologize for the confusion, the solution was already available (but in a different class) since version 4.0.0, but I forgot about it. 

(Available in version [4.0.6](https://github.com/ygimenez/Pagination-Utils/releases/tag/4.0.6))

To fix the issue you're having, simply change `setOnRemove()` in `PaginatorBuilder` to anything else (eg. an empty consumer). The default action is to remove the interactions from the message, changing it will allow you to do something else (or even leverage it as a way to get the events for your own handler).

```java
var paginator = PaginatorBuilder.createPaginator(/* Your client */)
        ... // Other settings
        .setOnRemove(hook -> {}) // Setting it as an empty consumer will suffice, or your can use it for other things.
        .build()
```
